.eslintrc.js

```diff
@@ -1,13 +1,6 @@
-const defaultConfig = require('./');
-
 module.exports = {
-  ...defaultConfig,
-  extends: [
-    // We do this filter to avoid our custom check for a prettier config on package.json
-    // for testing purposes
-    ...defaultConfig.extends.filter((config) => !config.includes('prettier')),
-    'plugin:prettier/recommended',
-  ],
+  extends: ['@bigcommerce/eslint-config'],
```

Comment:
This the new setup we don't need the previous hack. From the workspace root we can now directly extend a config even if it comes from a workspace package.